### PR TITLE
modify hamburger menu on small screen

### PR DESCRIPTION
**Pull request by Rishi007-web**
This is a small modification of the hamburger menu. So, please consider to add it.

### CODE REVIEW

**Comment by Rishi007-web** [comment]
Please review my code.

File: css/custom.css [hunk 1]
```diff
@@ -359,3 +359,80 @@ footer {
...
+    -webkit-transform: rotate(0deg);
+    -moz-transform: rotate(0deg);
+    -o-transform: rotate(0deg);
+    transform: rotate(0deg);
+    -webkit-transition: .5s ease-in-out;
+    -moz-transition: .5s ease-in-out;
+    -o-transition: .5s ease-in-out;
+    transition: .5s ease-in-out;
+    cursor: pointer;
+    }
```
Comment (jancborchardt) [inline css/custom.css]: All the closing parentheses don’t need to be indented. :)

File: index.html [hunk 2]
```diff
@@ -437,6 +437,12 @@ <h3>Programming Diversity</h3>
 		<script src="vendor/popper.min.js"></script>
 		<script src="vendor/bootstrap.min.js"></script>
 		<script>
+			        $(document).ready(function () {
+    		$('.first-button').on('click', function () {
+  
+			$('.animated-icon').toggleClass('open');
+    		});
+		});
```
Comment (jancborchardt) [inline index.html]: The indentation of this block seems off, could you fix that? :)

**Comment by Rishi007-web** [comment]
Thank You for your response.
Ok, I will start working on these changes very soon.

File: index.html (hunk 2 above)
Comment (Rishi007-web) [reply index.html]: Thank You for your response.
Ok, I will start working on these changes very soon.

Comment (Rishi007-web) [reply index.html]: Hello, I want to know one thing is I need to again clone this project and makes changes locally and then again PR or there is any other way of doing this.
Please guide me.

Comment (jancborchardt) [reply index.html]: Thanks, and sorry for the late reply! For the future, it’s always best to @-mention some people for review :)

Comment (jancborchardt) [reply index.html]: > Hello, I want to know one thing is I need to again clone this project and makes changes locally and then again PR or there is any other way of doing this.
> Please guide me.

You can simply add a commit to your fork and push it, the pull request should be updated. :)

Comment (Rishi007-web) [reply index.html]: OK, Thanks

Comment (Rishi007-web) [reply index.html]: Please tell me one more thing can I use vs code to indent it using alt+shift+f. This directly indents the code properly. and then I push the code. Or I do manually?

Comment (Rishi007-web) [reply index.html]: Please check it, I indent this block properly or not.

File: css/custom.css (hunk 1 above)
Comment (Rishi007-web) [reply css/custom.css]: I indented, please check it.

**Pull request merged**